components/divider/stories/divider.stories.js

```diff
@@ -45,5 +48,105 @@ export default {
 	},
 };
 
+/**
+ * The default size for divider is medium.
+ */
 export const Default = Template.bind({});
 Default.args = {};
+Default.parameters = {
+	docs: {
+		story: {
+			height: "auto",
+		}
+	},
+};
+
+/*
+ * Stories for the MDX "Docs" only.
+ * "storyName" refers to the display name/heading for a component
+ */
+const AllDividerSizes = (args) => html`
+	<div>
+		<div style="display: flex; flex-direction: column; padding: 16px;">
+			<h3 style="
+				margin: 0;
+				${styleMap({
+					"color": args.staticColor === "white" ? "rgb(255, 255, 255)": "inherit",
+				})}
+			">
+				Small
+			</h3>
+			${Template({
+				...args,
+				size: "s"
+			})}
+		</div>
+		<div style="display: flex; flex-direction: column; padding: 16px;">
+			<h3 style="
+				margin: 0;
+				${styleMap({
+					"color": args.staticColor === "white" ? "rgb(255, 255, 255)": "inherit",
+				})}
+			">
+				Medium (default)
+			</h3>
+			${Template({
+				...args,
+				size: "m"
+			})}
+		</div>
+		<div style="display: flex; flex-direction: column; padding: 16px;">
+			<h3 style="
+				margin: 0;
+				${styleMap({
+					"color": args.staticColor === "white" ? "rgb(255, 255, 255)": "inherit",
+				})}
+			">
+				Large
+			</h3>
+			${Template({
+				...args,
+				size: "l"
+			})}
+		</div>	
+	</div>
+`;
```

Comment:
I really like this as a way to display all the variants in one story! I think it's appropriate for divider because it's such a simple component but I wonder if it's a strategy we could use elsewhere 🤔

Reply:
Yeah definitely! I have it in drop indicator, and then in my other storybook docs branch, checkbox is using something similar (coach indicator too, I think). I had one for dial, but we decided not to use it since dial already had a bunch of stories represented.